libs/ngx-dsfr/tile/src/lib/testing/tile.component.spec.ts

```diff
@@ -0,0 +1,169 @@
+      By.directive(DsfrTileComponent)
+    ).componentInstance;
+
+    /*
+     * We're retrieving a HarnessLoader HERE to be able to get
+     * the Test Harness for the DsfrTileComponent LATER.
+     *
+     * We're NOT doing both HERE since getHarness() triggers
+     * indirectly the Change Detection mechanism and thus,
+     * Data Binding and ngOnInit()...
+     */
+    harnessLoader = TestbedHarnessEnvironment.loader(fixture);
+  });
+
+  it('should be created', () => {
+    expect(componentUnderTest).toBeTruthy();
+  });
+
+  describe('when required inputs are not provided, ', () => {
+
+    it('should throw an error when no title is provided', () => {
+      try {
+        fixture.detectChanges();
+        throw 'It should have thrown an error about "title"';
+      } catch (error) {
+        expect(error).toBe(EMPTY_TITLE_ERROR);
+      }
+    });
+
+    it('should throw an error when no link is provided', () => {
+      try {
+        testHost.testTitle = testTitle;
+        fixture.detectChanges();
+        throw 'It should have thrown an error about "link"';
+      } catch (error) {
+        expect(error).toBe(EMPTY_LINK_ERROR);
+      }
+    });
+  });
+
+  describe(' required inputs are provided, ', () => {
+    it('should display the right template when an external link is provided', async () => {
+      testHost.testLink = testExternalLink;
+      testHost.testTitle = testTitle;
+
+      /*
+       * We're retrieving the Test Harness HERE since we can now,
+       * without any problem, trigger the Change Detection mechanism
+       *
+       * WARNING: Triggers the Change Detection mechanism
+       */
+      dsfrTileHarness = await harnessLoader.getHarness<DsfrTileHarness>(
+        DsfrTileHarness
+      );
+      expect(await dsfrTileHarness.getLinkElementAttribute('href')).toBe(
+        testExternalLink
+      );
+      expect(await dsfrTileHarness.getLinkElementAttribute('target')).toBe('_blank');
+    });
+
+    it('should display the right template when an internal link is provided', async () => {
+
+      testHost.testLink = testInternalLink;
+      testHost.testTitle = testTitle;
+
+      /*
+       * We're retrieving the Test Harness HERE since we can now,
+       * without any problem, trigger the Change Detection mechanism
+       *
+       * WARNING: Triggers the Change Detection mechanism
+       */
+      dsfrTileHarness = await harnessLoader.getHarness<DsfrTileHarness>(
+        DsfrTileHarness
+      );
+
+      // Retrieving an instance of the RouterLinkDirectiveStub directive
+      const anchorWithRouterLink: DebugElement = fixture.debugElement.query(
+        By.directive(RouterLinkDirectiveStub)
+      );
+      const routerLinkStubInstance = anchorWithRouterLink.injector.get(
+        RouterLinkDirectiveStub
+      );
+      expect(routerLinkStubInstance.linkParams).toBe(testInternalLink);
+    });
+  });
+
+  describe(' all required properties are provided, ', () => {
```

Comment:
Actually, this `describe` and the one on line 96 are identical.
Given that everything is done properly in this one ( factorization of common lines in a `beforeEach` ), you should in fact put the two previous `it` in this `describe`...